### PR TITLE
update:improved readme and npm scripts

### PR DESCRIPTION
**Pull request by Waryserpant122**
I have added a script "dev" for dev purposed in case nodemon is not installed globally for the user. Also made necessary changes in the Readme. Please have a look and do tell me about it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,38 +95,52 @@ Following software needs to be setup in the system:
...
 ## Usage
+
 ### Local
+
 For running the project, navigate to the project directory and follow the given instructions:
 
-* Open terminal/cli and run:
-    ```sh
-    nodemon app.js
+- For running the development server,Open terminal/cli and run:
```
Comment (SubhradeepSS) [inline README.md:105]: the change can be => `For running the development server, open any terminal/cli and run:`

File: README.md [hunk 2]
```diff
@@ -92,38 +95,52 @@ Following software needs to be setup in the system:
...
+  ```sh
+    npm run dev
   ```
-    Open http://localhost:3000/ in browser and the login page of the project will open.
-* Create a user(by signing up from the site itself) with username: **admin** so as to give the user privileged permissions of an admin.
+- For tesing the server :
```
Comment (SubhradeepSS) [inline README.md:109]: change `For tesing the server :` => `For testing the server:`

**Comment by Waryserpant122** [comment]
I have made the changes as per the comments.

File: README.md (hunk 2 above)
Comment (SubhradeepSS) [reply README.md]: `tesing `=> `testing`

File: README.md [hunk 3]
```diff
@@ -92,38 +95,52 @@ Following software needs to be setup in the system:
...
-* Open terminal/cli and run:
-    ```sh
-    nodemon app.js
+- For running the development server, open terminal/cli and run:
+  ```sh
+    npm run dev
   ```
-    Open http://localhost:3000/ in browser and the login page of the project will open.
-* Create a user(by signing up from the site itself) with username: **admin** so as to give the user privileged permissions of an admin.
+- For tesing the server:
```
Comment (SubhradeepSS) [inline README.md:109]: `tesing `=> `testing`

Comment (Waryserpant122) [reply README.md]: Took care of this.

**Pull request merged**